server/protocols/jmap-rfc-8621/doc/specs/Makefile

```diff
@@ -0,0 +1,18 @@
+
+MMARK := /Users/neil/Code/Go/src/github.com/miekg/mmark/mmark/mmark -xml2 -page
```

Comment:
This line seems specific to a local env

Reply:
Will remove that makefile... it's from the jmap.io repo